### ISSUE TITLE
Wrong network-manager icon

### ISSUE THREAD

**Issue by Zeluboba** [numixproject/numix-icon-theme#674]
Hello,

today I ran apt-get update command and got wrong wired network-manager icon.
I found few similar issues, where it's said that it's fixed in 2014, but today is 2015 :)
I tried to reinstall as well.

![guszhhd](https://cloud.githubusercontent.com/assets/1494347/9478483/e009b206-4b80-11e5-8ddd-c4534e57721b.png)

**Comment by wa4557** [comment]
What DE are you using?

**Comment by Zeluboba** [comment]
@wa4557 pantheon

**Comment by bil-elmoussaoui** [comment]
Are you still using Luna ? (as im using Freya and i dont have this issue)

**Comment by Zeluboba** [comment]
@bil-elmoussaoui Luna.

**Comment by Foggalong** [comment]
If you're using Numix you should really use Freya. We've got a few bugs now which can't be fixed from our end but only affect Luna. I'd recommend if possible that you upgrade.

**Comment by Zeluboba** [comment]
@Foggalong It was fine before the apt-get update. And no, I'm sorry, I'm not going to update to Freya currently.

**Comment by Foggalong** [comment]
How long is it since you last did apt-get update?

(CC: @dirtydancing)

**Comment by Zeluboba** [comment]
@Foggalong yesterday

**Comment by Foggalong** [comment]
@Zeluboba That's really weird since [as you can see](https://github.com/numixproject/numix-icon-theme/commits/master) Numix hasn't received any updates for 5 days. Maybe this is an update to the wifi indicator package?

**Comment by Zeluboba** [comment]
@Foggalong no, numix package updated.
I have updated 26.08. So, why couldn't I receive that updates? And by the way I don't remember when I updated before it, so it could be month or so.

**Comment by Foggalong** [comment]
Couldn't receive which updates? Sorry I might be missing something :(

**Comment by Zeluboba** [comment]
@Foggalong so: last update: 26.08, pre-last: dunno, it could be month ago or more.

**Comment by wa4557** [comment]
To be honest, I wouldn't change anything concerning those icons. The thing is: we have come to a point where all major distributions work basically out of the box (even elementary (freya)); changing anything would require that we test all these DE again, and I'm almost 100% percent sure, that something else will break

**Comment by bil-elmoussaoui** [comment]
@Zeluboba can you paste the output of `dpkg -l | grep numix-*`?

**Comment by dirtydancing** [comment]
Huh, elementary OS Luna? I thought that one had reached end of life a while ago? I do not know why that icon is showing in the Luna panel (it should not). We would need to know the icon name of the displayed icon. If this is an unusual icon name for panel purposes, that might indicate a Luna problem.

**Comment by Foggalong** [comment]
@dirtydancing I spoke to Daniel recently about an issue relating to Numix and Luna technically doesn't reach EOL until 2017. Software will receive any needed security updates until then, it's just new features that won't be back ported to Luna.

With that in mind the recommendation is that we mark bugs that only affect Luna and don't have any solutions as wontfix.

**Comment by dirtydancing** [comment]
@Foggalong I see, thanks for clarifying about Luna :)

We will see about this issue. Experience just shows that elementary OS sometimes goes its own (inconsistent) way with regard to the panel icon naming, cf. e.g. the Bluetooth panel icon in Freya, https://github.com/numixproject/numix-icon-theme/issues/464.

**Comment by Zeluboba** [comment]
@bil-elmoussaoui will send you soon.
@wa4557 @dirtydancing @Foggalong I'm not going to update to Freya because there are a lot of bugs even in luna, I'm afraid what I will see in freya.

Well, I remember that icon was right from folder on screenshot, now it's wrong. I'll try to findcurrent icon name somewhere.

**Comment by Zeluboba** [comment]
@bil-elmoussaoui 

ii  elementary-enumix-utouch-icons            0.1-0~2~precise1                                eNumix-UTouch-Style Icon Theme for elementaryOS
ii  numix-gtk-theme                           1.9~precise                                     Numix GTK Theme
ii  numix-icon-theme                          0.3+512~201507241917~ubuntu12.04.1              Numix icon theme
ii  numix-icon-theme-circle                   1.0.2+4~201507161646~ubuntu12.04.1              Numix Circle icons

**Comment by Foggalong** [comment]
> elementary-enumix-utouch-icons

I don't think I've ever been so perturbed by a package name.

**Comment by Zeluboba** [comment]
@Foggalong I don't even have idea what is this :D

**Comment by Foggalong** [comment]
@Zeluboba [It's this](http://elementarytips.blogspot.co.uk/2013/07/enumix-utouch-icon-theme.html). Seems someone forked an old Numix icon theme and renamed it badly. Not really sure why :P

**Comment by Zeluboba** [comment]
@Foggalong @dirtydancing I couldn't find the icon in /usr/share/icons,
nothing about nm icon is found in syslog 8)
by the way, other statuses looks fine, like, not connected and connecting.

**Comment by Foggalong** [comment]
That's really weird! What does running `sudo apt-get remove elementary-enumix-utouch-icons` do?

**Comment by Zeluboba** [comment]
@Foggalong already removed and restarted both network-manager service and nm-applet.
By the way, the problem is same with both Numix and Numix-Circle.

**Comment by wa4557** [comment]
I guess it's `network-wired` or `nm-device-wired`, but again I guess I guess it's a luna specific bug

**Comment by Zeluboba** [comment]
@wa4557 @Foggalong 
found it!
It's "nm-device-wired.svg" in "/usr/share/icons/Numix/24x24/devices"

**Comment by wa4557** [comment]
Yeah this can't be removed or changed because of #461. But you could remove it for yourself :)

**Comment by Zeluboba** [comment]
@wa4557 I still don't understand why there are two different icon sets :|

**Comment by Foggalong** [comment]
@Zeluboba which two sets? :)

**Comment by wa4557** [comment]
circle and base, probably? Because it allows to be more flexible. There is for example not only circle but also square, having a seperate icon theme for all the non app-icons allows to use it with different app icon sets

**Comment by Zeluboba** [comment]
@Foggalong 
this one: http://i.imgur.com/UcM1LQ6.png
and this: http://i.imgur.com/qUhLUql.png

**Comment by Zeluboba** [comment]
Well, I deleted ones from /devices and now nm-applet doesn't have any icon 8) Should I make symlinks maybe? That's weird

**Comment by wa4557** [comment]
The icon should already be in `status` but sure, you can try to create a symlink

**Comment by Foggalong** [comment]
@Zeluboba some icons are used in device settings (the colourful ones) and some are used in the tray/panel (the white ones) :)

**Comment by dirtydancing** [comment]
@Zeluboba You might want to try deleting the icon cache and see if the proper icon then shows up.

If you have Numix installed globally, that would be (`-f` is for forcing the icon cache update; you might have to restart afterwards):

`sudo gtk-update-icon-cache -f /usr/share/icons/Numix`

**Comment by Foggalong** [comment]
@Zeluboba did any of that work? :)

**Comment by wa4557** [comment]
Btw.: @Zeluboba It seems that the package you are using is still outdated. Those icons from your first screenshot shouldn't be in circle anymore. Might be an issue with the ppa, maybe you should try to use the github version

**Comment by Foggalong** [comment]
Closing this as a wontfix. [See here](https://plus.google.com/b/115833527622591803679/+NumixProjectOrg/posts/FaTz9KSMvAu) for more info.

**Comment by Zeluboba** [comment]
@dirtydancing tried reinstalling the package and forced cache update. Didn't help.
@Foggalong there's 403 when I open your link :)
@wa4557 look the path, they're in 'numix', not numix-circle. Also I installed git version, the problem still remains. I decided to make the symlinks, but when I tried to find that files, "connect_no" for example, I didn't find any, lololo. I have no idea what's happened with my numix, but I don't like it really.
